pubspec.yaml

```
@@ -1,21 +1,20 @@
 name: reorderable_carousel
 description: A carousel in that allows one to reorder the contents of it via drag and drop 
-version: 0.3.2+1
+version: 0.4.0
```

I'll take care of the versioning
```suggestion
version: 0.3.2+1
```